docs/destinations/forwarding-events/google-tag-manager-server-side/launchdarkly-tag-for-gtm-ss/launchdarkly-tag-configuration/index.md

```diff
@@ -3,4 +3,120 @@ title: "LaunchDarkly Tag Configuration"
 sidebar_position: 100
 ---
 
-## Coming soon!
+## Event Name (Required)
+
+This is the value that identifies your metric. When you create your metric in LaunchDarkly, its Event name must exactly match this value.
+
+## Metric Type
+
+![metric-type](images/01-metric-type.png)
+
+This section allows you to select the Metric Type for this Tag. The options are:
+
+1. **Numeric Metric** (default): Choosing this metric type reveals the [Metric Options](#metric-options) configuration section, where you must specify a Metric Value.
+2. **Conversion**: Conversion Metrics should just be triggered and the metric name will be sent as a conversion.
+
+## Authentication
+
+![authentication](images/02-authentication.png)
+
+### Client Side ID (Required)
+
+In this text box you need to provide the client-side ID for the environment your metric events pertain to. In your LaunchDarkly account your client-side ID is available under Environments on the Projects tab on the Account settings page.
```

Comment:
Instead of giving the instructions directly, should we also link to the official docs ? _(In case they change something)_
E.g. https://docs.launchdarkly.com/sdk/concepts/client-side-server-side#client-side-id